src/main/webapp/skin-mobile-shared.xml

```diff
@@ -24,4 +24,6 @@
 
     <js>scripts/news.js</js>
 
+    <js included="plain" resource="true">/rs/handlebars/1.3.0/handlebars-v1.3.0.js</js>
+    <js included="aggregated" resource="true">/rs/handlebars/1.3.0/handlebars-v1.3.0.js</js>
```

Comment:
If you're using the same version of the file for both plain & aggregated, just use one element without the 'included' attribute.